src/libs/registry/docset.cpp

```diff
@@ -428,8 +429,11 @@ void Docset::countSymbols()
 // TODO: Fetch and cache only portions of symbols
 void Docset::loadSymbols(const QString &symbolType) const
 {
-    for (const QString &symbol : m_symbolStrings.values(symbolType))
-        loadSymbols(symbolType, symbol);
+    // itPair is a QPair<QMap::const_iterator, QMap::const_iterator>, with itPair.first and itPair.second respectively
```

Comment:
Nice, but comment is really unnecessary, because it just explains what is already clear from the documentation for `equal_range`.